pkgs/games/rott/default.nix

```diff
@@ -0,0 +1,60 @@
+{stdenv, lib, fetchurl, SDL, SDL_mixer, makeDesktopItem, copyDesktopItems, runtimeShell, buildShareware ? false}:
+
+let
+  desktopItem = makeDesktopItem {
+    name = "rott";
+    exec = "launch-rott";
+    desktopName = "Rise of the Triad: ${if buildShareware then "The HUNT Begins" else "Dark War"}";
+    categories = "Game;";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "rott";
+  version = "1.1.2";
+
+  src = fetchurl {
+    url = "https://icculus.org/rott/releases/${pname}-${version}.tar.gz";
+    sha256 = "1zr7v5dv2iqx40gzxbg8mhac7fxz3kqf28y6ysxv1xhjqgl1c98h";
+  };
+
+  buildInputs = [ SDL SDL_mixer copyDesktopItems ];
```

Comment:
Shouldn't `copyDesktopItems` be in `nativeBuildInputs`?

Reply:
ah good catch, I just fixed it